lib/rules/aria-roledescription.json

```diff
@@ -0,0 +1,13 @@
+{
+	"id": "aria-roledescription",
+	"selector": "[aria-roledescription]",
+	"excludeHidden": false,
+	"tags": ["cat.aria", "best-practice"],
+	"metadata": {
+		"description": "Ensures aria-roledescription attribute is used correctly",
+		"help": "Aria-roledescription attribute must be used on elements with an implicit or explicit role"
```

Comment:
The help text is supposed to be the shorter text. I also don't think this conveys quite enough information. Suggestion:

> description: Ensure aria-roledescription is only used on elements with an implicit or explicit role
> help: Use aria-roledescription on elements with a semantic role